basis/command.lua

```diff
@@ -16,6 +16,7 @@
 local S = function(pos) if pos then return minetest.pos_to_string(pos) end end
 --local P = minetest.string_to_pos
 --local M = minetest.get_meta
+local has_mesecons = minetest.get_modpath("mesecons_mvps") 
```

Comment:
better use `minetest.global_exists("mesecons_mvps")` 
If the mod `mesecons_mvps` is installed but not enabled, you code will crash.

Reply:
done